gemrb/core/Interface.cpp

```diff
@@ -2462,9 +2462,9 @@ int Interface::PlayMovie(const char* resref)
 
 			for (size_t i = 0; i < sttable->GetRowCount(); ++i) {
 				const char* frameField = sttable->QueryField(i, 0);
-				const char* strField = sttable->QueryField(i, 1);
-				if (frameField && strField) { // this skips the initial palette rows
-					subs[atoi(frameField)] = ieStrRef(atoi(strField));
+				ieStrRef strField = sttable->QueryFieldAsStrRef(i, 1);
+				if (frameField && strField != ieStrRef::INVALID) { // this skips the initial palette rows
```

Comment:
This is wrong, 0 is not -1.

Reply:
right, to me it seemed like a bug. why would it not be?

Reply:
That's not the value from the tables ...

Reply:
so is it a generality that tables contain 0 for undefined `ieStrRefs`? in other words should this be handled generally within `QueryFieldAsStrRef` where we can rewrite 0 to `ieStrRef::INVALID`

Reply:
No, tables don't have any type info or shared default value — it's per-table. Disabled strrefs are from memory usually -1 or -1 as unsigned, nothing we can be smart about. But in this particular case there's nothing invalid either, 0 is a real strref and the code needs to skip it.